hq/app/model/Serializers.scala

```diff
@@ -7,7 +7,15 @@ object Serializers {
 
   implicit val snykErrorFormat: Reads[SnykError] = Json.reads[SnykError]
 
-  implicit val snykOrgFormat: Reads[SnykOrganisation] = Json.reads[SnykOrganisation]
+  implicit val snykGroupFormat: Reads[SnykGroup] = Json.reads[SnykGroup]
+
+  implicit val snykOrgFormat: Reads[SnykOrganisation] = (
+    (JsPath \ "name").read[String]
+      and
+      (JsPath \ "id").read[String]
+      and
+      (JsPath \ "group").readNullable[SnykGroup]
+    )(SnykOrganisation.apply _)
```

Comment:
This one can probably be derived, but you'd need to call the property `group` (instead of `groupOpt`) so no worries if you prefer it this way